src/styles/global.tsx

```diff
@@ -4,7 +4,8 @@ import emotionReset from "emotion-reset";
 const style = css`
   ${emotionReset}
   html,
-  body {
+  body, #__next {
+    height: 100%;
```

Comment:
p5: height와 세트로   width도 한꺼번에 넣어주는 것은 어떨 지 왠지 글로벌로 height를 설정한 이유가 있으실 것 같아서요
```suggestion
    width:100%;
    height: 100%;
```

Reply:
footer를 하단에 고정하는 과정에서 height를 명시적으로 설정해줄 필요가 있었습니다! width는 예외가 없어서 꼭 명시해주지 않아도 되는 상황인데, 넣으면 좋겠다고 생각하시면 추가하도록 하겠습니당!